samples/web/content/apprtc/index.html

```diff
@@ -58,6 +58,11 @@
     </div>
   </div>
 
+  <div id="confirm-join-div" class="hidden">
+    <div> Are you ready to join <span id="confirm-join-room-span"></span>?</div>
```

Comment:
"Ready to join \"foo\"?" is shorter. Showing the room name in quotes helps make it obvious that this is the name of the room.

Also if the room name is one of our randomly-generated ones, I think we should simply omit it and display only "Ready to join?"